server/server-stats.cpp

```diff
@@ -475,6 +476,7 @@ struct WorkerProcessStats : private vk::not_copyable {
     misc_stats.inc_stat(MiscStat::Key::worker_activity_counter, worker_index);
     misc_stats.set_stat(MiscStat::Key::json_logs_count, worker_index, vk::singleton<JsonLogger>::get().get_json_logs_count());
     misc_stats.set_stat(MiscStat::Key::json_traces_count, worker_index, vk::singleton<JsonLogger>::get().get_json_traces_count());
+    misc_stats.set_stat(MiscStat::Key::threads_count, worker_index, get_self_threads_count());
   }
 
```

Comment:
you forgot to reset new stat here:
https://github.com/VKCOM/kphp/blob/a9aafd0bf159fa43675253fe336ddebad2b4943b/server/server-stats.cpp#L493

Reply:
done